phastpress.php

```diff
@@ -8,9 +8,56 @@
 License: AGPLv3
 */
 
-
+define('PHASTPRESS_IS_DEV', true);
 define('PHASTPRESS_PLUGIN_FILE', __FILE__);
 
+function phastpress_get_plugin_version() {
+    $plugin_info = get_file_data(PHASTPRESS_PLUGIN_FILE, ['Version' => 'Version']);
```

Comment:
With this change, the plugin will break on PHP 5.2. It shouldn't. (Square bracket arrays.)